tests/e2e/e2e_common.go

```diff
@@ -504,6 +504,7 @@ var (
 	envZone2DatastoreUrl                           = "ZONE2_DATASTORE_URL"
 	envIsolationSharedStoragePolicyNameLateBidning = "WORKLOAD_ISOLATION_SHARED_STORAGE_POLICY_WFFC"
 	envSharedZone1Zone2Zone3StoragePolicyName      = "SHARED_ZONE1_ZONE2_ZONE3_STORAGE_POLICY_IMM"
+	envAllZonalPolicyName                          = "ALL_ZONE_ZONAL_POLICY"
```

Comment:
env variable for zonal policies is already there , Please use the same  ZONAL_STORAGECLASS